tillerinobot/src/main/java/tillerino/tillerinobot/handlers/OptionsHandler.java

```diff
@@ -73,7 +73,13 @@ public int compare(LanguageIdentifier o1, LanguageIdentifier o2) {
 			} else {
 				ircUser.message("Welcome Message: " + (userData.isShowWelcomeMessage() ? "ON" : "OFF"));
 			}
-		} else {
+		} else if (getLevenshteinDistance(option, "recommendmods") <= 1) {
```

Comment:
I feel like this command should somehow communicate that it only applies as a default, i.e. when no more arguments are given.